src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -95,15 +95,23 @@ class ElectronicDeliveryForm extends React.Component {
               onChange={(e) => this.handleUpdate(e, 'emailAddress')}
             />
             {
-              errorClass.emailAddress &&
-                (<span
+              (errorClass.emailAddress) ?
```

Comment:
For all the checks you do, the `<span>` is always there and the only thing that changes is the text that gets displayed. You could have the span there and do `(errorClass.emailAddress) ? {this.state.error.emailAddress} : Please enter a valid email address` instead. Not just for field but for the rest too where it gets rendered twice.